index.html

```diff
@@ -222,8 +235,63 @@ <h3>tccc mode</h3>
     <p>Overlapping more than two notes is undefined behavior in tccc mode. Don't do it.</p>
   </div>
 
+  <div id="pitchbend" class="container">
+    <h2>Pitch Bend Events</h2>
+
+    <p>TCCC can perform additional adjustments to the pitch of notes using MIDI Pitch Bend events, allowing you to 
+      set notes to microtonal values.</p>
+
+    <ul>
+      <li>In your DAW, MIDI pitch bend events are a numerical value that is converted to a change in pitch 
+        by the output device (e.g. your synthesizer plugin, soundfont player, etc).</li>
+      <ul>
+        <li>Tutorials if you are unfamiliar with MIDI pitch bends:</li>
+        <ul>
+          <li>Reaper: 
+            <a href="https://www.youtube.com/watch?v=4rOsXnQSoT8">Federico Bonacossa</a></li>
+          <li>Ableton: 
+            <a href="https://quadrophone.com/synthesis/using-pitch-bend/">Using Pitch Bend - Quadrophone</a></li>
+          <li>FL Studio: 
+            <a href="https://macprovideo.com/article/audio-software/fl-studio-automation-clips-vs-channel-editing-vs-per-note-editing">
+            Macprovideo (use the Channel Editing method)</a></li>
+        </ul>
+        <li>Most output devices default to +-2 semitones, and some allow you to change this, but it varies across 
+            devices so it's best to check your own setup.</li>
+      </ul>
+      <br>
+      <li>In TCCC, the Pitch Bend Range option controls what range (in semitones) the MIDI events are converted to.</li>
+      <ul>
+        <li>E.g. At a Pitch Bend Range of 2, a maximum value pitch bend MIDI event will raise/lower the note pitch by 2 semitones.</li>
+        <li>Make sure this range matches the range of the DAW output device you used when making the MIDI.</li>
+      </ul>
+      <br>
+      <li>TCCC takes the pitch bend value at the start of a particular note, and shifts the entire note by that amount.</li>
+    </ul>
+
+      <h3>Examples</h3>
+    <p>Pitch bend event covers all notes, so all are shifted up by the same amount (the unadjusted note is shown faded out).</p>
+    <img src="res/pitch_example_both_notes.png"/>
+
+    <p>Pitch bend event covers just the first note of the slide, this note is shifted down.</p>
+    <img src="res/pitch_example_first_note.png"/>
+
+    <p>You can shift the start/end of a single-note slide by placing the pitch bend around just the start/end note. In this case the end
+      of the slide is shifted down.</p>
+    <img src="res/pitch_example_single_slide.png"/>
+
+    <p> When shifting a note connected to the end of a slide, make sure to shift both the end note and the slide note that connects to it.</p>
+    <img src="res/pitch_example_second_note.png"/>
+
+    <p>For pitch shift gradients/curves, the pitch shift amount is taken at the start point of each note.</p>
```

Comment:
Could this be confusing to people if the pitch bend events don't apply to the tail of each note as well? Like, shouldn't the notes in the image all be slightly upward slides?

Reply:
Given this some thought and I think you're right, users would expect the note ends to be a different pitch if the pitch bend is different to the start of the note (also this allows for stuff like making slides purely out of pitch bends). 

I'll need to update the generation functions to support this, also I'll need to update the rest of the tutorial as it changes best practices. In particular in this situation the end of the note will now be lower because the last pitch event on the note end time is zero:
![image](https://github.com/TC-Chart-Converter/TC-Chart-Converter.github.io/assets/135999125/1ea06ad0-3e20-46bd-b4f3-e6a235caeec1)

If users want to ensure the entire note has the same pitch then the pitch bend would need to end after the note like this:
![image](https://github.com/TC-Chart-Converter/TC-Chart-Converter.github.io/assets/135999125/9043e8ec-3345-4ab6-b9d9-9496af975d83)

Reply:
Updated note generation and tutorial: c6745acfe353ba194f60a47e548da83353aeedf